File: tests/arch-constraint/fixtures/python/bad_import.py
"""Fixture: запрещённый импорт из соседнего модуля платформы.

Этот файл сам по себе не валиден — используется только runner'ом
скрипта `check-arch-constraint.sh` для проверки detection'а.
"""

# Нарушение 1: from rehome_kb_platform.X import Y
from rehome_kb_platform.api.articles import Article

# Нарушение 2: import kb_search (без `from`)
import kb_search

__all__ = ["Article", "kb_search"]

File: tests/arch-constraint/fixtures/typescript/bad_import.ts
// Fixture: запрещённый импорт из соседнего frontend-модуля.
// Должен быть detect'ен скриптом AT-001.

import { Article } from "rehome-kb-platform/articles";
import { fetchKb } from "../kb-search/client";

export const x = { Article, fetchKb };
